docs/json/radarr/cf/51-surround.json

```diff
@@ -13,7 +13,7 @@
       }
     },
     {
-      "name": "6.1 Surround",
+      "name": "Not 6.1 Surround",
       "implementation": "ReleaseTitleSpecification",
```

Comment:
Keep the 6.1 negate out of the 5.1 CF. I think it can go either way (delete the 5.1 negate in the 6.1CF and add 6.1 negate in 5.1 CF), but my gut says there is a reason it was missing since we started with every negate in every CF. It must have been intentionally removed at some point